hypersync-format/src/types/bloom_filter_wrapper.rs

```diff
@@ -25,6 +26,29 @@ impl FilterWrapper {
     pub fn insert_hash(&mut self, hash: u64) -> bool {
```

Comment:
IMO we should remove this method and make it immutable, they should construct it at once, so only can implement from_keys method for construction

Reply:
I was thinking it might be useful in HyperIndex to maintain one filter and add to it when new dynamic contracts are registered

Reply:
We would get janky false positive rate if it works like this

Reply:
Removed :+1: